libtalek/client_test.go

```diff
@@ -109,13 +110,19 @@ func TestRead(t *testing.T) {
 
 	//Due to thread race, there may be a random read made before
 	//the requested poll is queued up.
-	decRead1, _ := read1.Decode(0, config.TrustDomains[0])
-	decRead2, err := read2.Decode(0, config.TrustDomains[0])
+	decRead11, _ := read1.Decode(0, config.TrustDomains[0])
```

Comment:
Based on my reading of it, it this broke based on the final change i asked you to make in #78 - when you switched the libtalek to use the common bucket encoding the 1 for the chosen bucket isn't always set in the request to the first trust domain anymore.